### ISSUE TITLE
[BUG] framer-motion-3d MotionCanvas does not resize/update with parent container

### ISSUE THREAD

**Issue by 3lijah** [motiondivision/motion#1444]
**1. Read the [FAQs](#faqs) 👇**

**2. Describe the bug**

When resizing the element containing the canvas, the canvas should resize as well.

**3. IMPORTANT: Provide a CodeSandbox reproduction of the bug**

In this r3f example, the element stays centered when resizing the iframe:
https://codesandbox.io/s/react-spring-animations-6hi1y

In this framer-motion-3d example, the element retains its initial coordinates:
https://codesandbox.io/s/framer-motion-3d-variants-9173p

**4. Steps to reproduce**

1. Load a page that contains a MotionCanvas that takes up the full window size.
2. Resize the window
3. Notice that the canvas does not resize to fit the new window size.

**5. Expected behavior**

The canvas should resize with its parent and the scene should update to reflect the new size and aspect ratios.

**6. Video or screenshots**

![image](https://user-images.githubusercontent.com/1908949/152394208-a83c09bf-e166-4a25-902a-5d41cfd91105.png)

**Comment by KimPaow** [comment]
@3lijah did you figure out what the cause of this is? The docs mention that resize is omitted from the original canvas component, but nothing more, I guess this should work out of the box.

**Comment by 3lijah** [comment]
> @3lijah did you figure out what the cause of this is? The docs mention that resize is omitted from the original canvas component, but nothing more, I guess this should work out of the box.

No, I haven't made any progress. It does seem intentional but problematic for full-window experiences.

**Comment by KimPaow** [comment]
From what I can understand there simply doesnt appear to be any implementation for resizing:  [source](https://github.com/framer/motion/blob/main/packages/framer-motion-3d/src/components/MotionCanvas.tsx#L104-L116)

The [r3f canvas](https://github.com/pmndrs/react-three-fiber/blob/master/packages/fiber/src/web/Canvas.tsx#L50) uses [use-measure](https://github.com/pmndrs/react-use-measure) which responds to changes. Maybe i'm missing something, but to me this seems like an easy thing to implement.

**Comment by wolfgangschoeffel** [comment]
Does anybody have a solution for this? Is there a way to trigger the resize handler somehow from outside of the component?

**Comment by sdacunha** [comment]
Sadly seeing this still, if anyone has a workaround, it would be greatly appreciated.
